Src/NHibernate.Envers.Tests/Integration/Basic/TransactionScopeDistributed.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Transactions;
+using NHibernate.Envers.Configuration;
+using NHibernate.Envers.Tests.Entities;
+using NUnit.Framework;
+
+namespace NHibernate.Envers.Tests.Integration.Basic
```

Comment:
In NHibernate.Envers.Tests.Integration there is converted Hibernate Envers tests, so these tests suits better in NetSpecific.Integration instead.